### ISSUE TITLE
Save last column value to last_run_metadata_path more frequently

### ISSUE THREAD

**Issue by science695** [logstash-plugins/logstash-input-jdbc#295]
If logstash has an error or something causes the pipeline to end prematurely, the pipeline has to start from the previous saved column value.

If you are running an initial import (or have a significant amount of data) in the pipeline's execution, it can take a while for it to completely finish importing your rows. It can be very frusterating to have logstash import for several hours and then error out, and when you fix the error and start it up again it has to start all over again.

Feature Request: Can jdbc save the last column value to the last_run_metadata_path file more frequently? Perhaps after each jdbc_page_size?

That way if there was an outage, or error while a large pipeline was running, it could resume the job from the last page at least.

- Operating System: CentOS 7.5
- Docker 1.13.1
- Logstash image version 6.2.3

**Comment by science695** [comment]
Yes, I pondered this sorting problem myself as I was writing my queries for logstash.

Could this be a setting that is documented to need the results to be sorted by tracking column? (I understand why you wouldn't want to parse the SQL to find the orderby).

Or, you could make an order_by option for users to put their order by clause in, and then logstash would know if it was sorted or not.

**Comment by vara855** [comment]
++ Must have feature

**Comment by jacksparrow414** [comment]
Any new progress?
